### PR TITLE
Update Vietnamese, Initial Japanese and Korean language

### PR DESCRIPTION
**Pull request by dungphanxuan**

### CODE REVIEW

File: config/crater.php [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
         ["code"=>"de", "name" => "German"],
+        ["code"=>"ja", "name" => "Japanese"],
         ["code"=>"it", "name" => "Italian"],
         ["code"=>"lv", "name" => "Latvian"],
         ["code"=>"pt_BR", "name" => "Portuguese (Brazilian)"],
         ["code"=>"sr", "name" => "Serbian Latin"],
+        ["code"=>"ko", "name" => "Korean"],
         ["code"=>"es", "name" => "Spanish"],
         ["code"=>"sv", "name"=> "Svenska"],
-        ["code"=>"sk", "name"=> "Slovak"],
```
Comment (mohitpanjwani) [inline config/crater.php:25]: Hey, @dungphanxuan: I think you've removed the Slovak language from this list. 

We should keep that as it is and add the new languages above.

File: resources/assets/js/plugins/i18n.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ import fr from './fr.json'
...
 import de from './de.json'
+import ja from './ja.json'
 import pt_BR from './pt-br.json'
 import it from './it.json'
 import sr from './sr.json'
 import nl from './nl.json'
+import ko from './ko.json'
 import lv from './lv.json'
 import sv from './sv.json'
-import sk from './sk.json'
```
Comment (mohitpanjwani) [inline resources/assets/js/plugins/i18n.js:14]: Same thing here, I think you might have removed this language while trying to fix the conflict.

**Review by mohitpanjwani — CHANGES_REQUESTED** [review]

File: resources/assets/js/plugins/i18n.js (hunk 2 above)
Comment (dungphanxuan) [reply resources/assets/js/plugins/i18n.js:14]: yes, i have update it. thank you!

**Comment by mohitpanjwani** [comment]
Hi @dungphanxuan, 

Awesome work! Thanks a lot for your contribution. 

Although I cannot understand Korean/Japanese/Vietnamese this is really great, I'm sure it'll be super helpful for others who are more comfortable with these languages.

I'm curious though, are you familiar with all 3 languages? 

Merging this PR btw ✅  

Thanks again

**Pull request merged**